opencda/scenario_testing/utils/sim_api.py

```diff
@@ -187,6 +196,20 @@ def __init__(self, scenario_params,
         # set weather
         weather = self.set_weather(simulation_config['weather'])
         self.world.set_weather(weather)
+        # Define probabilities for each type of blueprint
+        self.use_multi_class_bp = scenario_params[
+            'use_multi_class_bp'] if 'use_multi_class_bp' in scenario_params else False
+        if self.use_multi_class_bp:
+            # bbx/blueprint meta
+            with open(scenario_params['bp_meta_path']) as f:
```

Comment:
we didn't provide any template of sample for the prob file, users won't have any idea how to write such a json. please provide a sample at least and put under asset folder

Reply:
have added one.